File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex15.java
import java.util.Scanner;

class Ex15 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite 3 lados de um triangulo");
		double side1 = scan.nextDouble();
		double side2 = scan.nextDouble();
		double side3 = scan.nextDouble();

		if (side1 + side2 > side3 ||
			side1 + side3 > side2 ||
			side2 + side3 > side1) {

			if (side1 == side2 &&
				side1 == side3 &&
				side2 == side3) {
				System.out.println("Triangulo Equilatero");
			} else if ((side1 == side2 || side1 == side3) && 
					   (side2 == side3 || side2 == side1)) {
				System.out.println("Triangulo Isosceles");
			} else {
				System.out.println("Triangulo Escanleno");
			}
		} else {
			System.out.println("Os lado indicados nao formam um triangulo");
		}
		scan.close();
	}
}

File: Loiane_Training_C#/Basico/C/Ex15.cs
using System;

public class Ex15
{
    static void Main(string[] args)
    {
        Console.WriteLine("Informe os 3 lados de um triangulo");
        double CatetoOposto = double.Parse(Console.ReadLine());
        double CatetoAdjascente = double.Parse(Console.ReadLine());
        double Hipotenusa = double.Parse(Console.ReadLine());

        if (CatetoAdjascente + CatetoOposto > Hipotenusa ||
            Hipotenusa + CatetoAdjascente > CatetoOposto ||
            CatetoOposto + Hipotenusa > CatetoAdjascente &&
            CatetoAdjascente > 0 && CatetoOposto > 0 && Hipotenusa > 0)
        {
            if (CatetoAdjascente == CatetoOposto && CatetoAdjascente == Hipotenusa)
            {
                Console.WriteLine("Triangulo Equilatero");
            }
            else if (CatetoAdjascente == CatetoOposto || CatetoAdjascente == Hipotenusa)
            {
                Console.WriteLine("Triangulo Isosceles");
            }
            else
            {
                Console.WriteLine("Triangulo Escanleno");

            }
        }
        else
        {
            Console.WriteLine("Numeros informados nao formam um triangulo!");
        }
    }
}
